airsonic-main/src/main/webapp/WEB-INF/jsp/sonoslinkClose.jsp

```diff
@@ -0,0 +1,23 @@
+<%@ page language="java" contentType="text/html; charset=utf-8" pageEncoding="iso-8859-1"%>
+<!DOCTYPE html>
+
+<html><head>
+    <%@ include file="head.jsp" %>
+    <script type="text/javascript">
+        if (window != window.top) {
+            top.location.href = location.href;
+        }
+    </script>
+
+</head>
+<body style="min-width:550px" class="mainframe bgcolor1" onload="document.getElementById('j_username').focus()">
```

Comment:
Please use the [autofocus]( https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autofocus )  property directly on the input field instead of using javascript.

Reply:
I copied/pasted the code from the login page and left it as such so as not to mix styles too much.